src/spyglass/position/v1/dlc_utils.py

```diff
@@ -421,7 +421,6 @@ def get_video_path(key):
     from ...common.common_behav import VideoFile
 
     vf_key = {"nwb_file_name": key["nwb_file_name"], "epoch": key["epoch"]}
-    VideoFile()._no_transaction_make(vf_key, verbose=False)
```

Comment:
My understanding is that this line was previously included to ensure that the key was generated prior to fetching. I don't mind removing and having folks hit the print below more often, but I'd rather address whatever circularity could be happening on the other side of the call. 

@bsharma-hiprep - do you have the error stack available? Maybe we can address the infinite loop at the source?

Reply:
@CBroz1 is there any reason to not check if `vf_key` is present in the table and only insert if not?